README.md

```diff
@@ -232,6 +232,36 @@ vars:
 ### Override the data models variable
 This package is designed to run **either** the Netsuite.com or Netsuite2 data models. However, for documentation purposes, an additional variable `netsuite_data_model_override` was created to allow for both data model types to be run at the same time by setting the variable value to `netsuite`. This is only to ensure the [dbt docs](https://fivetran.github.io/dbt_netsuite/) (which is hosted on this repository) is generated for both model types. While this variable is provided, we recommend you do not adjust the variable and instead change the `netsuite_data_model` variable to fit your configuration needs.
 
+### Lookback Window
+Records from the source can sometimes arrive late. Since several of the models in this package are incremental, by default we look back 3 days to ensure late arrivals are captured while avoiding the need for frequent full refreshes. While the frequency can be reduced, we still recommend running `dbt --full-refresh` periodically to maintain data quality of the models. 
+
+To change the default lookback window, add the following variable to your `dbt_project.yml` file:
+
+```yml
+vars:
+  netsuite:
+    lookback_window: number_of_days # default is 3
+```
+
+### Adding incremental materialization for Bigquery and Databricks
+Due to the variation in pricing and runtime priorities for customer, by default we chose table instead of incremental materialization for Bigquery and Databricks. For more information on this decision, see the [Incremental Strategy section](https://github.com/fivetran/dbt_netsuite/blob/main/DECISIONLOG.md#incremental-strategy) of the DECISIONLOG.
+
+If you wish to enable it, you can add the below materialization settings to your `dbt_project.yml` file. You only need to add lines for the specific model materializations you wish to change.
+```yml
+models:
+  netsuite:
+    netsuite2:
+      netsuite2__income_statement:
+        +materialized: incremental # default is table for Bigquery and Databricks
+      netsuite2__transaction_details:
+        +materialized: incremental # default is table for Bigquery and Databricks
+      netsuite2__balance_sheet:
+        +materialized: incremental # default is table for Bigquery and Databricks
+      intermediate:
+        int_netsuite2__tran_with_converted_amounts:
+          +materialized: incremental # default is ephemeral for Bigquery and Databricks
```

Comment:
When running the models with this setup I get the following error. Any thoughts as to what this error is a result of?

<img width="1262" alt="image" src="https://github.com/fivetran/dbt_netsuite/assets/74217849/4c37d7f6-a0f1-4280-bfa7-2816d981c39b">

Reply:
Then if I only change the materialization of the end models (ignoring the intermediate model) I get the following error.

<img width="1262" alt="image" src="https://github.com/fivetran/dbt_netsuite/assets/74217849/00a3008a-33fd-4c27-a643-ba0a18cc5fad">

Reply:
What variables and/or data were you using? If your selections produce duplicates, then the merge won't work. The variables I put in my validation should allow this to work, but let me know if it still doesn't. @fivetran-joemarkiewicz

Reply:
@fivetran-catfritz ahhh okay let me try your variables. Is this something you feel we need to callout directly?

Reply:
2nd update--after our discussion and deep dive, changing the default behavior is not necessary as we were just seeing artifacts from our own data. Reverting default behavior back to the original false.